alf/networks/encoding_networks.py

```diff
@@ -281,39 +282,47 @@ def __init__(self,
         from the other layers.
```

Comment:
Remove "create an ..." template for all class init functions

Reply:
Updated according to the comment.